### PR TITLE
Feature/trees lfg/#3

### LINKED ISSUE #3: Train POC Trees
DoD:

* train dependency parser on created trees (which are already split on train/test/develop
* share test set metrics on slack

### PR DESCRIPTION
**Pull request by Gizzio**
Plz make sanity check on dep parser training (low accuracy on POS) closes #3

### CODE REVIEW

**Comment by kowaalczyk** [comment]
The process looks fine, POS accuracy is bad due to overfitting - 6th epoch of learning seemed to still be fine:
```bash
$ cat models/trees-pos_LFG_word2vec/model5/accuracy.json
{
  "uas":90.6647673314,
  "las":85.584045584,
  "ents_p":0.0,
  "ents_r":0.0,
  "ents_f":0.0,
  "tags_acc":85.0362457077,
  "token_acc":100.0
}

$cat models/trees-pos_LFG_word2vec/model6/accuracy.json
{
  "uas":90.5879000855,
  "las":85.706144933,
  "ents_p":0.0,
  "ents_r":0.0,
  "ents_f":0.0,
  "tags_acc":28.920259443,
  "token_acc":100.0
}
```

It might be worth to explore reason for overfitting POS tags in greater detail, especially given that the POS tag accuracy is dropping much sooner than the dependency parser metrics and the fact that validation was performed on the training test itself.

I suggest training the model with a proper validation now that the train-test splitting script is present in the master branch. After rebase, training and evaluation the branch can be merged.

Also, always remember about marking that it closes #3

**Comment by Gizzio** [comment]
Thanks for checking. I performed validation on dev data set, LFG is already split into test, train and dev. I trained it again on  

The reason of overfitting POS sooner than dependency parser is worth exploring. My hypothesis is that it is because of nature of metrics used.
